source/Phage.h

```diff
@@ -233,6 +273,44 @@ class Phage: public Symbiont {
     sym_baby->mutate();
     return sym_baby;
   }
+
+  /**
+   * Input: location of the phage attempting to horizontally transmit
+   * 
+   * Output: None
+   * 
+   * Purpose: To allow lytic/induced phage to lyse their host 
+   */
+  void HorizontalTransmission(size_t location) { //Horizontal transmission process after lysis or induction are called
```

Comment:
I'm not convinced that HorizontalTransmission is the best name for this method. Isn't it just about Lysis (since there are other ways that horizontal transmission can happen with phage theoretically)? Or maybe it should actually be two different methods (LysisBurst and LysisStep maybe?) and the burst timer if statement could stay in process. 

The main concern I have is that the code in Process is confusing since it reads "not lysogeny, so horizontal transmission" which hides too many of the details of what is actually going on. It should read "not lysogeny, so check the burst timer and either lyse or make another offspring for eventual lysis"